tests/test_models.py

```diff
@@ -38,3 +37,52 @@ def test_updated(self, session, Article):
         session.commit()
 
         assert article.updated >= then and article.updated <= datetime.utcnow()
+
+
+class TestGenericReprMixin:
```

Comment:
Rename this test case (its testing generic_repr not GenericReprMixin so maybe TestGenericRepr would be good).